zeth-contracts/contracts/BaseMixer_tests.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.5.0;
+
+import './BaseMixer.sol';
+
+contract BaseMixer_tests {
+	using Bytes for *;
+
+	constructor() public {
+        // Nothing
+    }
+
+    function test_extract_extra_bits() public returns (bool) {
+        // test_bytes = primary inputs
```

Comment:
Please make sure to clean up the indentation in the files of the repo

Reply:
Done